sdk/core/Azure.Core/samples/Diagnostics.md

```diff
@@ -94,12 +150,14 @@ Azure SDKs are instrumented for distributed tracing using ApplicationsInsights o
 
 Application Insights, a feature of Azure Monitor, is an extensible Application Performance Management (APM) service for developers and DevOps professionals. Use it to monitor your live applications. It will automatically detect performance anomalies, and includes powerful analytics tools to help you diagnose issues and to understand what users actually do with your app
 
-If your application already uses ApplicationInsights, automatic collection of Azure SDK traces is supported since version `2.12.0`. 
+If your application already uses ApplicationInsights, automatic collection of Azure SDK traces is supported since version `2.12.0` ([Microsoft.ApplicationInsights on NuGet](https://www.nuget.org/packages/Microsoft.ApplicationInsights/)).
 
 To setup ApplicationInsights tracking for your application follow the [Start Monitoring Application](https://docs.microsoft.com/azure/azure-monitor/learn/dotnetcore-quick-start) guide.
 
 ### OpenTelemetry with Azure Monitor, Zipkin and others
 
+OpenTelemetry relies on ActivitySource to collect distributed traces. Follow steps in [ActivitySource support](#ActivitySource support) section before proceeding to OpenTelemetry configuration.
+
 Follow the [OpenTelemetry configuration guide](https://github.com/open-telemetry/opentelemetry-dotnet#configuration-with-microsoftextensionsdependencyinjection) to configure collecting distribute tracing event collection using the OpenTelemetry library.
 
```

Comment:
Can we add something for potential tracing vendors or advanced users looking for the details on what we emit? Something along the lines of 

"If you want to process, filter, or query Azure SDK telemetry spans or build Azure-specific visualizations, here are the [semantic conventions](https://github.com/Azure/azure-sdk/blob/main/docs/tracing/distributed-tracing-conventions.yml) Azure SDK follows."

Reply:
Can do. The only thing I worry about is the mixed messaging. Do we treat https://github.com/Azure/azure-sdk/blob/main/docs/tracing/distributed-tracing-conventions.yml as experimental?

Reply:
We're not going to change them ever, but we're going to switch to otel conventions and stop supporting ours. 
They are also our contract with AzMon until OTel stabilizes, they are versioned, follow semver, and didn't change in the past year or two.

So I'd say they are not experimental, but we'll have a new major version in the next year switching to something standard hopefully everyone supports by then.

Reply:
it makes me think that we should include schema_url of that spec on the spans, so processing could be versioned

Reply:
>So I'd say they are not experimental, but we'll have a new major version in the next year switching to something standard

Yeah, that sounds like a breaking change and I don't really want customers to start depending on something we expect to remove in a year.

Reply:
Ok, so do you think we shouldn't mention them at all or say they're to play with but not depend upon?